kolibri_server_setup.py

```diff
@@ -66,6 +66,16 @@ def disable_cherrypy():
     """
     update_options_file('Server', "CHERRYPY_START", False, KOLIBRI_HOME)
 
+
+def enable_cherrypy():
+    """
+    Disables internal kolibri web server.
+    Kolibri will only run background tasks.
+    Web must be provided by an external server, usually uwsgi + nginx
+    """
```

Comment:
The comments need to be negated as they seem apt for disable_cherrypy. May be a copy/paste oversight.

Reply:
Fixed, thanks!